src/System.Runtime.Extensions/src/System/AppDomain.cs

```diff
@@ -26,5 +27,147 @@ public partial class AppDomain
+        {  
+            add  
+            {  
+                AppDomainAugments.ProcessExit += value;  
+            }  
+  
+            remove  
+            {  
+                AppDomainAugments.ProcessExit -= value;  
+            }  
+        }  
+
+        public void AppendPrivatePath(string path) { throw new PlatformNotSupportedException(); }
+        public string ApplyPolicy(string assemblyName) => AppDomainAugments.ApplyPolicy(assemblyName);
+        public void ClearPrivatePath() { throw new PlatformNotSupportedException(); }
+        public void ClearShadowCopyPath() { throw new PlatformNotSupportedException(); }
+        public static System.AppDomain CreateDomain(string friendlyName) { throw new PlatformNotSupportedException(); }
+        public int ExecuteAssembly(string assemblyFile) => ExecuteAssembly(assemblyFile, null, null, System.Configuration.Assemblies.AssemblyHashAlgorithm.None);
+        public int ExecuteAssembly(string assemblyFile, string[] args) => ExecuteAssembly(assemblyFile, args, null, System.Configuration.Assemblies.AssemblyHashAlgorithm.None);
+        public int ExecuteAssembly(string assemblyFile, string[] args, byte[] hashValue, System.Configuration.Assemblies.AssemblyHashAlgorithm hashAlgorithm)
+        {
+            AppDomainAugments.ExecuteAssembly(assemblyFile, args, hashValue, hashAlgorithm); 
+        }
+        public int ExecuteAssemblyByName(System.Reflection.AssemblyName assemblyName, params string[] args) => AppDomainAugments.ExecuteAssemblyByName(assemblyName, args);
+        public int ExecuteAssemblyByName(string assemblyName) => ExecuteAssemblyByName(assemblyName, null);
+        public int ExecuteAssemblyByName(string assemblyName, params string[] args) => AppDomainAugments.ExecuteAssemblyByName(assemblyName, args);
+        public object GetData(string name) => AppContext.GetData(name);
+        // Should this move to AppContext
+        public void SetData(string name, object data) => AppDomainAugments.SetData(name, data);
+        public System.Nullable<bool> IsCompatibilitySwitchSet(string value) => AppDomainAugments.IsCompatibilitySwitchSet(value);
+        public bool IsDefaultAppDomain()  => true;
+        public bool IsFinalizingForUnload() => false;
+        public void SetCachePath(string path) { throw new PlatformNotSupportedException(); }
+        public void SetPrincipalPolicy(System.Security.Principal.PrincipalPolicy policy) { throw new PlatformNotSupportedException(); }
+        public void SetShadowCopyFiles() { throw new PlatformNotSupportedException(); }
+        public void SetShadowCopyPath(string path) { throw new PlatformNotSupportedException(); }
+        public void SetThreadPrincipal(System.Security.Principal.IPrincipal principal) { throw new PlatformNotSupportedException(); }
+        public override string ToString() => AppDomainAugments.ToString();
+        public static void Unload(System.AppDomain domain) { throw new PlatformNotSupportedException(); }
```

Comment:
Doesn't desktop throw `CannotUnloadAppDomainException` if trying to unload the default domain? Should this do the same?